pkg/role_binding/role_binding.go

```diff
@@ -153,3 +172,24 @@ func PresentRoleBindings(roleBindings []*v1.RoleBinding) []AccountRoleBinding {
 	})
 	return uniqueRoleList
 }
+
+func PresentSubscriptionRoleBindings(roleBindings []*v1.RoleBinding) []SubscriptionsRoleBinding {
+	var subscriptionRoleBindings []SubscriptionsRoleBinding
+	uniqueRoleBindingsMap := make(map[string][]string)
+	for _, roleBinding := range roleBindings {
+		if uniqueAccountRoleBindings, ok := uniqueRoleBindingsMap[roleBinding.Account().ID()]; ok {
+			uniqueAccountRoleBindings = append(uniqueAccountRoleBindings, roleBinding.Role().ID())
+			uniqueRoleBindingsMap[roleBinding.Account().ID()] = uniqueAccountRoleBindings
+		} else {
+			uniqueAccountRoleBindings := []string{roleBinding.Role().ID()}
+			uniqueRoleBindingsMap[roleBinding.Account().ID()] = uniqueAccountRoleBindings
```

Comment:
As discussed, I am not sure if grouping by AccountID will be very useful. Let's take a look in a few examples from the stage/integration environment to determine the best grouping strategy.